image/Dockerfile

```diff
@@ -31,6 +31,10 @@ RUN /development/qt5/configure -nomake examples -nomake tests -prefix /usr/local
 RUN cmake --build . $EXTRA_BUILD_PARAMS
 RUN cmake --install .
 
+# Apply WASM specific patch
```

Comment:
Nitpick: Can you change it to mention that the app will be included in the Qt 6.6 release, so that it's clear that it's a temporary patch.

Reply:
Added a comment about it